src/diffpy/morph/morphs/morphsqueeze.py

```diff
@@ -9,9 +9,29 @@ class MorphSqueeze(Morph):
 
     Configuration Variables
     -----------------------
-    squeeze
-        list or array-like
-        Polynomial coefficients [a0, a1, ..., an] for the squeeze function.
+    squeeze : list
+        The polynomial coefficients [a0, a1, ..., an] for the squeeze function
+        where the polynomial would be of the form a0 + a1*x + a2*x^2 and so
+        on.  The order of the polynomial is determined by the length of the
+        list.
+
+    Example
+    -------
+    >>> import numpy as np
+    >>> from numpy.polynomial import Polynomial
+    >>> from diffpy.morph.morphs.morphsqueeze import MorphSqueeze
+
+    >>> x_target = np.linspace(0, 10, 101)
```

Comment:
Let's remove this block of text because it will confuse people.  This is us building a target, but the assumption is that the user already has a target, so none of this code is needed by people just to use the morph.